ingestion/src/metadata/utils/life_cycle_utils.py

```diff
@@ -80,3 +82,23 @@ def get_query_type(create_query) -> Optional[str]:
         logger.debug(traceback.format_exc())
         logger.warning(f"Unexpected exception get the query type: {exc}")
     return None
+
+
+def patch_life_cycle(
+    metadata: Optional[OpenMetadata], entity: Entity, life_cycle: LifeCycle
+) -> None:
+    """
+    Patch life cycle data for a entity
+
+    :param entity: Entity to update the life cycle for
+    :param life_cycle_data: Life Cycle data to add
+    """
+    try:
+        destination = entity.copy(deep=True)
+        destination.lifeCycle = life_cycle
+        metadata.patch(entity=type(entity), source=entity, destination=destination)
+    except Exception as exc:
+        logger.debug(traceback.format_exc())
+        logger.warning(
+            f"Error trying to Patch life cycle data for {entity.fullyQualifiedName.__root__}: {exc}"
+        )
```

Comment:
Patching part would be a better fit in mixins rather than utils

Reply:
let's use utils for helper functions

Reply:
Wanted to put it in mixins, but in mixins we use the `REST` client. Here I want to directly call the `metadata.patch` method from mixins itself.
This is brought here only so that I can do the deep copy for the patch API

Reply:
we have similar thing done for `patch_test_case_definition` under patch_mixins

Reply:
makes sense I'll move it under patch_mixins then